### PR TITLE
Add `--file_name=<name>` argument for `wp media import`

### PR DESCRIPTION
**Pull request by sandeshjangam**
Added a slug option for media

Issue - https://github.com/wp-cli/media-command/issues/186

### CODE REVIEW

**Review by vrundakansara — APPROVED** [review]

File: src/Media_Command.php [hunk 1]
```diff
@@ -185,6 +185,9 @@ public function regenerate( $args, $assoc_args = array() ) {
 	 * [--post_name=<post_name>]
 	 * : Name of the post to attach the imported files to.
 	 *
+	 * [--slug=<slug>]
```
Comment (danielbachhuber) [inline src/Media_Command.php:188]:
```suggestion
	 * [--file_name=<name>]
```

Let's use `file_name` as the argument, for greater clarity.

**Review by danielbachhuber — CHANGES_REQUESTED** [review]
Thanks for the pull request, @sandeshjangam !

Can you include some functional tests? Here's the documentation on how to do so: https://make.wordpress.org/cli/handbook/contributions/pull-requests/#functional-tests

**Comment by danielbachhuber** [comment]
> Can you include some functional tests? Here's the documentation on how to do so:

@sandeshjangam If it's helpful, here's a video I put together too: https://github.com/wp-cli/wp-cli/issues/5858

File: src/Media_Command.php (hunk 1 above)
Comment (sandeshjangam) [reply src/Media_Command.php]: DOne

**Comment by sandeshjangam** [comment]
@danielbachhuber Done with the functional test. Thanks for the reference.

File: src/Media_Command.php [hunk 2]
```diff
@@ -414,6 +423,10 @@ public function import( $args, $assoc_args = array() ) {
 			}
 
 			$attachment_success_text = '';
+			if ( $assoc_args['file_name'] ) {
+				$attachment_success_text .= ", file name {$name}";
```
Comment (schlessera) [inline src/Media_Command.php:427]:
```suggestion
				$attachment_success_text .= " with file name {$name}";
```

**Review by schlessera — CHANGES_REQUESTED** [review]

**Review by danielbachhuber — APPROVED** [review]
Thanks for your work on this, @sandeshjangam !

**Pull request merged**